docs/helpers/ObjectStorage.md

```diff
@@ -18,6 +18,14 @@ The Object Storage Helper will help you handle storage of generic objects within
 > [!div class="nextstepaction"]
 > [Try it in the sample app](uwpct://Helpers?sample=Object%20Storage)
 
+## Constructors
+
+LocalObjectStorageHelper and RoamingObjectStorageHelper have the same constructor with an optional parameter:
+
+| Constructor | Description |
+|-------------|-------------|
+| ObjectStorageHelper(IObjectSerializer = null) | Initializes a new instance of the ObjectStorageHelper class with the provided IObjectSerializer, that will be used when serializing and deserializing data in Settings or in Storage; if no serializer is provided, a default Json serializer will be used. |

```

Comment:
```suggestion
| ObjectStorageHelper(IObjectSerializer = null) | Initializes a new instance of the ObjectStorageHelper class with the provided IObjectSerializer, that will be used when serializing and deserializing data in Settings or in Storage; if no serializer is provided, a default JSON serializer will be used. |

```

Reply:
Also the IObjectSerializer here should link to the other ObjectSerializer doc (and placed in the API section below) (which also should be added to the `TOC.md`). @simop-msft